NGitLab/Models/SnippetUpdate.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
+
+namespace NGitLab.Models
+{
+    public class SnippetUpdate
+    {
+        [Required]
+        [JsonPropertyName("id")]
+        public int SnippedId;
```

Comment:
Use getter and setter for new properties.

```suggestion
        public int SnippetId { get; set; }
```